homework/index.js

```diff
@@ -30,14 +32,145 @@
     return elem;
   }
 
+  function openNewTab(url) {
+    window.open(url, '_blank');
+  }
+
+  function bindContributions(contributorsUrl) {
+    fetchJSON(contributorsUrl, (error, response) => {
+      const repoInfoContainer = document.getElementById('repo-info-container');
+      const contributorsDiv = createAndAppend('div', repoInfoContainer, {
+        class: 'contributors-info',
+      });
+      if (error) {
+        createAndAppend('div', contributorsDiv, {
+          text: error.message,
+          class: 'alert-error',
+        });
+        return;
+      }
+
+      const div = createAndAppend('div', contributorsDiv);
+      createAndAppend('h4', div, { text: 'Contributions' });
+      createAndAppend('hr', div);
+      const table = createAndAppend('div', div, { class: 'grid' });
```

Comment:
The contributors list is best expressed as a list (for ARIA-compliance). Visually impaired users that use a screen reader will rely on your HTML to follow the semantic rules, e.g. a list must be presented by an HTML list, in this case a `ul`. Each item in the list should be an `li`.